packages/delivery.yaml

```diff
@@ -10,49 +10,49 @@ definitions:
     tags_regex: [^master(-experiment)?$]
     constant_tags: [nightly] # add new tags.
   sync_trunk_enterprise: &sync_trunk_enterprise {}
-  sync_rc_community: &sync_rc_community
+  sync_pre_community: &sync_pre_community
     description: delivery community images for release branches.
     tags_regex:
       - ^release-[0-9]+[.][0-9]+(-beta[.][0-9]+)?$
       - ^v[0-9]+[.][0-9]+[.][0-9]+-pre$
-      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+([.]pre)?$
+      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+[.]pre$
       - ^v[0-9]+[.][0-9]+[.][0-9]+-rc[.][0-9]+[.]pre$
-  sync_rc_community_and_failpoint: &sync_rc_community_and_failpoint
+  sync_pre_community_and_failpoint: &sync_pre_community_and_failpoint
     description: delivery community and failpoint images for release branches.
     tags_regex:
       - ^release-[0-9]+[.][0-9]+(-beta[.][0-9]+)?(-failpoint)?$
       - ^v[0-9]+[.][0-9]+[.][0-9]+-pre(-failpoint)?$
-      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+([.]pre)?(-failpoint)?$
+      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+[.]pre(-failpoint)?$
       - ^v[0-9]+[.][0-9]+[.][0-9]+-rc[.][0-9]+[.]pre(-failpoint)?$
-  sync_rc_enterprise: &sync_rc_enterprise
+  sync_pre_enterprise: &sync_pre_enterprise
     description: delivery the enterprise RC images.
     tags_regex:
       - ^(v[0-9]+[.][0-9]+[.][0-9]+-pre)-enterprise$
       - ^(v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+[.]pre)-enterprise$
       - ^(v[0-9]+[.][0-9]+[.][0-9]+-rc[.][0-9]+[.]pre)-enterprise$
     tag_regex_replace: "$1"
-  sync_rc_community_to_enterprise_repo: &sync_rc_community_to_enterprise_repo
+  sync_pre_community_to_enterprise_repo: &sync_pre_community_to_enterprise_repo
     description: delivery the community RC image to enterprise repos.
     tags_regex:
       - ^v[0-9]+[.][0-9]+[.][0-9]+-pre$
-      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+([.]pre)?$
+      - ^v[0-9]+[.][0-9]+[.][0-9]+-beta[.][0-9]+[.]pre$
```

Comment:
Why does it need to be adjusted here to only match this format v9.0.0-beta.1-pre?

Reply:
the beta.1 artifacts will be delivered as normal GA version.